### PR TITLE
- Task40

### PR DESCRIPTION
**Pull request by artsiomMandrukevich**
`LogInTest` is a parametrized test for log in.
`MultiSelectTest` is tests for multiselect
`AllertsTest` is tests for alerts
`UserWaiting` is an automated script, which waits for use
`RefreshPage` is an automated script, which refreshes the page when percentage is >= 50
`EmployeesFilter` is an automated script, which returns a list of custom objects with `age > x` and `salary <= y` from Web Table

### CODE REVIEW

File: src/test/java/helpers/Element.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package helpers;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class Element {
```
Comment (LikBez123) [inline src/test/java/helpers/Element.java]: remove this class

File: src/test/java/AllertsTest.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        driver.manage().window().maximize();
+        driver.get("https://demo.seleniumeasy.com/javascript-alert-box-demo.html");
+    }
+
+    @Test
+    public void alertBox() {
+        element.clickButton(driver, ALERT_BOX_BUTTON);
+
```
Comment (LikBez123) [inline src/test/java/AllertsTest.java]: remove empty lines

File: src/test/java/MultiSelectTest.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    public void startDriver() {
+        WebDriverManager.getInstance(ChromeDriver.class).setup();
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        driver.manage().window().maximize();
+    }
+
+    @Test
+    public void multiSelect() {
+        driver.get("https://demo.seleniumeasy.com/basic-select-dropdown-demo.html");
```
Comment (LikBez123) [inline src/test/java/MultiSelectTest.java:33]:
```
@Test
    public void multiSelect() {
        driver.get("https://demo.seleniumeasy.com/basic-select-dropdown-demo.html");
        Select multiSelect = new Select(driver.findElement(MULTI_SELECT));
        List<String> expectedStates = new Random().ints(3, 0, multiSelect.getOptions().size()).boxed()
                .map(i -> multiSelect.getOptions().get(i).getText()).collect(Collectors.toList());
        expectedStates.forEach(state -> multiSelect.selectByValue(state));
        List<String> selectedStates = multiSelect.getAllSelectedOptions().stream().map(element -> element.getText())
                .collect(Collectors.toList());
        assertTrue(expectedStates.containsAll(selectedStates));
    }
```

**Review by LikBez123 — CHANGES_REQUESTED** [review]

File: src/test/java/MultiSelectTest.java (hunk 3 above)
Comment (artsiomMandrukevich) [reply src/test/java/MultiSelectTest.java:33]: done

File: src/test/java/AllertsTest.java (hunk 2 above)
Comment (artsiomMandrukevich) [reply src/test/java/AllertsTest.java]: done

File: src/test/java/helpers/Element.java (hunk 1 above)
Comment (artsiomMandrukevich) [reply src/test/java/helpers/Element.java]: done

**Review by LikBez123 — APPROVED** [review]

**Pull request merged**